integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -1109,6 +1109,40 @@ public static void callShellScriptByExecToPod(
     logger.info("Command to call kubectl sh file " + cmdKubectlSh);
     TestUtils.exec(cmdKubectlSh.toString());
   }
+  
+  /**
+   * exec into the pod and call the shell script with given arguments.
+   *
+   * @param podName pod name
+   * @param domainNS namespace
+   * @param scriptsLocInPod script location
+   * @param shScriptName script name
+   * @param args script arguments
+   * @throws Exception exception
+   */
+  public static void callShellScriptInsidePod(
```

Comment:
can you see if you can use the existing method callShellScriptByExecToPod in TestUtils

Reply:
sure

Reply:
Ok, I firstly tried that method but it turned out that "-R" in below line didn't work for me
.append(" -- bash -c 'chmod +x -R ")
In my case I can only chmod for that specific file, not recursively.

Reply:
I see..you can try creating a directory under /u01/oracle and see if that works with callShellScriptByExecToPod..if not you can refactor callShellScriptByExecToPod itself for your needs, for example add another argument boolean changeDirPermissions - if true use -R

```
  public static void checkDb(String[] dburl, String rcuPod) throws Exception {
  	String scriptsLocInPod = "/u01/oracle/scripts";
        TestUtils.exec("mkdir -p "+scriptsLocInPod , true);
  	TestUtils.copyFileViaCat(
        BaseTest.getProjectRoot() + "/integration-tests/src/test/resources/jrf/checkDb.sh",
        scriptsLocInPod + "/checkDb.sh",
        rcuPod,
        DEFAULT_RCU_NAMESPACE);
    TestUtils.callShellScriptInsidePod(
        rcuPod, DEFAULT_RCU_NAMESPACE, scriptsLocInPod, "checkDb.sh", dburl);
```